modules/genomics_nextflow/orgpolicy.tf

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+resource "google_project_organization_policy" "external_ip_policy" {
```

Comment:
Any reason why we need to enable this Org policy as we are not setting External IP to the VM ?

Reply:
This might be needed if the user decides not to use Internal IPs only for worker nodes. But I'm happy to remove it.